fastlane/Fastfile

```diff
@@ -839,3 +892,26 @@ platform :ios do
     UI.message("Gutenberg version: #{(res.scan(/'([^']*)'/))[0][0]}")
   end
 end
+
+def read_version_from_config
+  fastlane_require 'Xcodeproj'
+
+  # If the file is not available, the method will raise so we should be fine
+  # not handling that case. We'll never return an empty string.
+  File.open(File.join(PROJECT_ROOT_FOLDER, 'Config', 'Version.public.xcconfig')) do |config|
+    configuration = Xcodeproj::Config.new(config)
+    configuration.attributes['VERSION_SHORT']
+  end
+end
+
+# Application-agnostic settings for the `upload_to_app_store` (also known as
+# `deliver`) action.
```

Comment:
I should really stop being nitpicky 😅  but I know you like it and have similar OCD as me so I know you'll indulged me 😸 
```suggestion
# Application-agnostic settings for the `upload_to_app_store`
# (also known as `deliver`) action.
```

Reply:
Ahahah! Yeah, I can see how that might be visually bothering.

I format long lines using my text editor and it would restore it to that form next time I asked it to format the line, so I decided to changed the wording to remove the parenthesis in 8ed1baa0cd 🤷‍♂️